Makefile

```diff
@@ -116,16 +116,19 @@ freeze_db: ## Freeze the Meltano database to support DB migration during upgrade
 
 # sdist:
 # Build the source distribution
-# Note: plese use `sdist-public` for the actual release build
+# Note: please use `sdist-public` for the actual release build.
+# Note: despite being called "sdist", this builds both the sdist and wheel.
 sdist: freeze_db bundle ## Build the Meltano sdist for development
```

Comment:
Can `freeze_db` and `bundle` be removed?

Reply:
`freeze_db` updates `db.lock`, which is included in the wheel so maybe that has to run as part of the build too?

I don't know if that's only relevant for `meltano upgrade`, though.

Reply:
I've added a step to the GitHub workflow that previously called `sdist_public` to freeze the DB. With that done, it no longer has any reason to use the Makefile, so I replaced the call to `make sdist_public`. Now nothing in the repository relies on the Makefile.